deep_quoridor/src/train_dqn.py

```diff
@@ -32,11 +32,8 @@ def train_dqn(
     """
     game = env(board_size=board_size, max_walls=max_walls, step_rewards=step_rewards)
 
-    # Calculate action space size
-    action_size = board_size**2 + ((board_size - 1) ** 2) * 2
-
     # Create the DQN agent
-    dqn_agent = FlatDQNAgent(board_size, action_size)
+    dqn_agent = FlatDQNAgent(board_size, epsilon_decay=0.9999)
```

Comment:
@alejandromarcu and this is the other bug you found, this attempts to make it decay a bit slower

Reply:
nit, remove the line above computing action_size.
Also, I'm wondering if the epsilon decay should happen after episode rather than after training step